### PR TITLE
Install GitHub CLI on Windows images

### PR DESCRIPTION
**Pull request by dsame**
# Description
New tool
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  

Installation size: 16.5Mb
Installation time:  5 sec

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-GitHub-CLI.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+################################################################################
+##  File:  Install-GitHub-CLI.ps1
+##  Desc:  Install GitHub CLI
+################################################################################
+
+
+Choco-Install -PackageName gh
+
+Import-Module -Name ImageHelpers -Force
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Install-GitHub-CLI.ps1:9]: I think we should move Import Module to the begin of the file

Comment (dsame) [reply images/win/scripts/Installers/Install-GitHub-CLI.ps1]: I cloned this code from the others scripts, but i agree it is more convenient to have all imports on top

File: images/win/scripts/Installers/Validate-GitHub-CLI.ps1 [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+else
+{
+    Write-Host 'gh is not on path'
+    exit 1
+}
+
+# Adding description of the software to Markdown
+$SoftwareName = "GitHub CLI"
+
+$version = (gh --version) -replace 'gh version (\d+\.\d.\d).*', '$1'
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-GitHub-CLI.ps1:21]: (gh --version).Split()[3]

**Pull request merged**